### DISCUSSION TITLE
Length dengan Interval tidak sesuai pada pengecekan All

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#616] [Q&A] [answered]
Dear ka @kevbow 
saya cukup kesulitan ketika melihat tanggal nya yang diambil adalah tanggal akhir bulan, sudah saya pakaikan if else tetapi bentrok dengan bulan februari. cukup sulit untuk di sesuaikan. apakah ada solusi ?
solusi yang terpikirkan dari saya sendiri adalah manipulate data dari akhir bulan ke awal bulan, tapi yang saya takutkan adalah data tidak sesuai dengan kebutuhan bisnis berjalan..
![image](https://user-images.githubusercontent.com/123348110/230271323-fab890e1-a867-40a9-8f98-99614cff25fa.png)

![image](https://user-images.githubusercontent.com/123348110/230271508-b253b748-6b77-4e5c-a8c0-ae3c5f2b7055.png)

**Comment by Yusraf002** [comment]
Dear @DeanAlexander27  ,
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak.

**Comment by kevbow** [comment, marked as answer]
Dear @DeanAlexander27,

Solusi yang Anda sampaikan sudah tepat karena informasi yang ada hanyalah data bulanan sehingga jika diambil insightnya adalah insight bulanan. 
Silakan sampaikan alasan ini secara naratif sebagai penjelasan Anda bahwa data tersebut merupakan data di akhir bulan dan proses yang dilakukan merupakan pengecekan berdasarkan syarat teknikal pembuatan time series.

Demikian yang dapat saya sampaikan, terima kasih.
